### DISCUSSION TITLE
Table 组件列顺序支持客户端存储与服务器端存储

### DISCUSSION THREAD

**Discussion by A5196060** [dotnetcore/BootstrapBlazor#3323] [Q&A] [answered]
现在的应用有时候需要表格列顺序持久化到本地，用户调整后关闭浏览器，再打开仍然保持最后一次列顺序

同理，有时候需要序列化到服务器端，更换电脑登录后，仍然保持上一次的列顺序

**Comment by densen2014** [comment, marked as answer]
那你得把排序存到数据库，每个用户记忆还得分id存一遍就行
